src/components/NcAppNavigationToggle/NcAppNavigationToggle.vue

```diff
@@ -45,6 +45,11 @@ import { t } from '../../l10n.js'
 import MenuIcon from 'vue-material-design-icons/Menu.vue'
 import MenuOpenIcon from 'vue-material-design-icons/MenuOpen.vue'
 
+/**
+ * The NcAppNavigationToggle component is used typically alongside another header navigation component.
+ * It emits the status of whether the button has been clicked or not.
+ *
+ */
```

Comment:
It is supposed to be used only in `<NcAppNavigation>` and is not exported from the library.

We should probably remove it from the docs instead.